x-pack/test/api_integration/apis/security/change_password.ts

```diff
@@ -74,21 +94,26 @@ export default function ({ getService }: FtrProviderContext) {
 
       const newSessionCookie = cookie(passwordChangeResponse.headers['set-cookie'][0])!;
 
-      // Let's check that previous cookie isn't valid anymore.
+      // Old cookie is still valid (since it's still the same user and cookie doesn't store password).
```

Comment:
__note:__ it's something that we should change and invalidate session ID on password change. But I don't have a good solution yet since we both want to invalidate old session to regenerate new SID and at the same time keep all existing session properties like `accessAgreementAcknowledged`. Do you think we can do that in a follow-up?

Reply:
Yeah I think this is fine to address in a follow-up